### ISSUE TITLE
Move event-specific CSS into plugin

### ISSUE THREAD

**Issue by joedolson** [wpa11yday/wpaccessibilityday#5]
Since we're moving to having one permanent site about WP Accessibility Day in general and a separate site for each year's event, any CSS that's specific to the event info should be moved into the plugin.

**Comment by amberhinds** [comment]
Don't judge me... or do because I totally deserve it, but I put styles in the customizer as we got really close to the event and I was afraid to push changes.

There is probably CSS there that needs to be moved into the plugin.

**Comment by joedolson** [comment]
@lefty55104 This is probably done, but if you can search through the remaining CSS in style.css to find anything that only applies to event data (sponsors, speakers, sessions, schedule, etc.), that would be great. Anything you find, you can split into /css/event.css and I'll deal with merging into the plugin CSS.

**Comment by lefty55104** [comment]
@joedolson Looks like all of the customizer code has been copied into the theme, so it should be fine to remove. I didn't want to remove it without making sure the theme was updated, though. 

There were a few page-specific css that I removed since they no longer exist, and there's some CSS in there that I couldn't tell where it belonged, but I left it since it didn't say anything specifically. 

Changes are pushed to event-specific-css branch.
